tests/hmm/hmumu_utils.py

```diff
@@ -245,6 +245,12 @@ def analyze_data(
         use_cuda
     )
 
+    # event-by-event mass resolution
+    dpt1 = (leading_muon["ptErr"]*higgs_inv_mass) / (2*leading_muon["pt"])
+    dpt2 = (subleading_muon["ptErr"]*higgs_inv_mass) / (2*subleading_muon["pt"])
+    higgs_inv_mass_uncertainty = NUMPY_LIB.sqrt(dpt1*dpt1 + dpt2*dpt2)
+    higgs_rel_inv_mass_uncertainty = higgs_inv_mass_uncertainty / higgs_inv_mass
```

Comment:
If I understand correctly, you also define these calculations later in the `compute_fill_dnn` function: https://github.com/jpata/hepaccelerate-cms/pull/32/files#diff-7c681c3161b87061604d79dc8695e4ffR1937
If it's OK with you, can we just use the output of the function? Just to avoid redefining the calculation twice.